poh/benches/poh.rs

```diff
@@ -65,3 +74,73 @@ fn bench_poh_lock_time_per_batch(bencher: &mut Bencher) {
         poh.hash(DEFAULT_HASHES_PER_BATCH);
     })
 }
+
+#[bench]
+fn bench_poh_recorder_record_transaction_index(bencher: &mut Bencher) {
+    let ledger_path = get_tmp_ledger_path_auto_delete!();
+    bencher.iter(|| {
```

Comment:
there's a bunch in here that doesn't really make sense to time. Only the function/code we want to benchmark should be inside ther `iter` call - all the set up should be outside.